packages/lighthouse-spa/README.md

```diff
@@ -0,0 +1,30 @@
+# Lighthouse SPA
+Lighthouse SPA is the interface test a web property with support of lighthouse ci.
```

Comment:
Can you make it more descriptive? This doesn't tell what the SPA exactly is.

Reply:
Updated the description